Framework/scripts/getSamplesCfg.sh

```diff
@@ -3,8 +3,11 @@
 cfgDir=${CMSSW_BASE}/src/Framework/Framework/cfg
 
 #Default options
-sampleSCfgFile="sampleSets_v6.cfg"
-sampleCCfgFile="sampleCollections_v6.cfg"
+#sampleSCfgFile="sampleSets_v6.cfg"
+#sampleCCfgFile="sampleCollections_v6.cfg"
+# use v5 for stop gen level study
+sampleSCfgFile="sampleSets_v5.cfg"
```

Comment:
Don't push this change. It will break our code. Also, you should be using V6 now anyway.